settings-patch.php

```diff
@@ -0,0 +1,299 @@
+					'description' => __( 'Break comments into pages', 'wp-graphql' ),
+					'default' => false
+				),
+				'comments_per_page' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Number of top level comments per page', 'wp-graphql' )
+				),
+				'default_comments_page' => array(
+					'show_in_rest' => array(
+						'schema' => array(
+							'enum' => array( 'newest', 'oldest' ),
+						),
+					),
+					'type' => 'string',
+					'description' => __( 'Comment page displayed by default', 'wp-graphql' )
+				),
+				'comment_order' => array(
+					'show_in_rest' => array(
+						'schema' => array(
+							'enum' => array( 'asc', 'desc' ),
+						),
+					),
+					'type' => 'string',
+					'description' => __( 'Order comments are displayed in', 'wp-graphql' )
+				),
+				'comment_registration' => array(
+					'show_in_rest' => true,
+					'type' => 'boolean',
+					'description' => __( 'Users must be registered and logged in to comment', 'wp-graphql' ),
+					'default' => false
+				),
+				'show_comments_cookies_opt_in' => array(
+					'show_in_rest' => true,
+					'type' => 'boolean',
+					'description' => __( 'Show comments cookies opt-in checkbox', 'wp-graphql' )
+				),
+			),
+			'media' => array(
+				'thumbnail_size_w' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum width for thumbnail-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				'thumbnail_size_h' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum height for thumbnail-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				'thumbnail_crop' => array(
+					'show_in_rest' => true,
+					'type' => 'boolean',
+					'description' => __( 'Crop thumbnail to exact dimensions (normally thumbnails are proportional)', 'wp-graphql' )
+				),
+				'medium_size_w' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum width for medium-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				'medium_size_h' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum height for medium-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				'large_size_w' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum width for large-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				'large_size_h' => array(
+					'show_in_rest' => true,
+					'type' => 'integer',
+					'description' => __( 'Maximum height for large-sized content in pixels to use when adding an image to the Media Library', 'wp-graphql' )
+				),
+				/**
+				 * TODO: Get more info on these three options
```

Comment:
@kidunot89 I'll have to look into this as well 🤔

Reply:
@kidunot89 looks like `image_default_size` should be an Enum of registered image sizes, eh? 🤔